Sources/web3swift/Web3/Web3+HttpProvider.swift

```diff
@@ -39,4 +39,14 @@ public class Web3HttpProvider: Web3Provider {
         }
         attachedKeystoreManager = manager
     }
+    
+    public init(
+        url: URL,
+        network: Networks,
+        keystoreManager: KeystoreManager? = nil
+    ) {
```

Comment:
These style doesn't match with all the other code in our project, please format it in more swift way, like init attributes should be in one line and open curly brackets should follow them

Reply:
done https://github.com/web3swift-team/web3swift/pull/717/commits/e893cb37f377fdb7eb283265a90cfb86f07d1bf2

Reply:
@yaroslavyaroslav For large long functions and inits I'd break them into multiple lines.
E.g. for the `init?` of `Web3HttpProvider`:
```
public init?(_ httpProviderURL: URL,
             network net: Networks? = nil,
             keystoreManager: KeystoreManager? = nil) async {
                 ...
             }
```

instead of what we currently have.

@rinat-enikeev Nothing to change for you in this PR, just raising commenting on this topic.